src/cascadia/TerminalSettingsEditor/Launch.xaml

```diff
@@ -56,44 +65,58 @@ the MIT License. See LICENSE in the project root for license information. -->
-                                        Margin="0,0,0,20"
-                                        FontSize="15"
-                                        SpinButtonPlacementMode="Compact" 
-                                        SmallChange="10"
-                                        LargeChange="100"
-                                        ToolTipService.Placement="Mouse"/>
-                <Controls:NumberBox x:Uid="Globals_InitialRows"
-                                    x:Name="InitialRows"
-                                        Value="{x:Bind GlobalSettings.InitialRows, Mode=TwoWay}"
-                                        Margin="0,0,0,20"
-                                        FontSize="15"
-                                        SpinButtonPlacementMode="Compact" 
-                                        SmallChange="10"
-                                        LargeChange="100"
-                                        ToolTipService.Placement="Mouse"/>-->
+                <CheckBox x:Uid="Globals_StartOnUserLogin" IsChecked="{x:Bind State.Settings.GlobalSettings.StartOnUserLogin, Mode=TwoWay}"/>
+                <Controls:RadioButtons x:Uid="Globals_LaunchSize"
+                                       SelectedItem="{x:Bind CurrentLaunchSize}"
+                                       ItemsSource="{x:Bind LaunchSizeList}"
+                                       ItemTemplate="{StaticResource EnumRadioButtonTemplate}"/>
+
+                <Grid Margin="0,0,0,20">
+                    <Grid.Resources>
+                        <Style TargetType="TextBlock">
+                            <Setter Property="FontSize" Value="15"/>
+                            <Setter Property="VerticalAlignment" Value="Center"/>
+                            <Setter Property="Margin" Value="0,0,5,0"/>
+                        </Style>
+                    </Grid.Resources>
+                    <Grid.RowDefinitions>
+                        <RowDefinition Height="auto"/>
+                        <RowDefinition Height="auto"/>
+                    </Grid.RowDefinitions>
+                    <Grid.ColumnDefinitions>
+                        <ColumnDefinition Width="auto"/>
+                        <ColumnDefinition Width="auto"/>
+                    </Grid.ColumnDefinitions>
+
+                    <TextBlock x:Uid="Globals_InitialSize"
+                               Grid.Row="0" Grid.Column="0"
+                               Grid.ColumnSpan="2"/>
+
+                    <StackPanel Orientation="Horizontal"
+                                Grid.Row="1" Grid.Column="0"
+                                Margin="0,0,10,0">
+                        <TextBlock x:Uid="Globals_InitialCols"/>
+                        <Controls:NumberBox Value="{x:Bind State.Settings.GlobalSettings.InitialCols, Mode=TwoWay}"
+                                            SmallChange="10"
+                                            LargeChange="100"

```

Comment:
nit: 100 is like, a _huge_ difference in terminal size - maybe these should be 1 and 10?